lib/krikri/parser.rb

```diff
@@ -216,6 +262,10 @@ def self.build(record)
       def get_field(name)
         self.class.new(flat_map { |val| val[name] })
       end
+
+      public
+
+      class InvalidParserValueError < StandardError; end
```

Comment:
Should this subclass `TypeError`? That might be a little more appropriate, but I don't have a strong feeling either way.